Sources/AsyncHTTPClient/ConnectionPool.swift

```diff
@@ -562,7 +562,7 @@ class HTTP1ConnectionProvider {
                     error = HTTPClient.NWErrorHandler.translateError(error)
                 }
             #endif
-            return self.eventLoop.makeFailedFuture(error)
+            return eventLoop.makeFailedFuture(error)
```

Comment:
think this is the wrong way around, no?

Reply:
No, I think this is correct, all channel related stuff should be on channel EL, not on providers default EL

Reply:
@artemredkin sorry, I meant from `self.` to missing `self`, `eventLoop` isn't a local variable, is it?

Reply:
it is, yes, this is a channel EL, I can rename it to make it more obvious

Reply:
@artemredkin yes, I think that'd be a good idea tbh